concepts/authorities.py

```diff
@@ -164,3 +165,13 @@ def get_by_namespace(namespace):
 
     return [ manager for manager in authority_managers
                 if manager.namespace == namespace ]
+
+def add_concept(instance):
+    """
+    Adding the approved concept to Conceptpower
+    :param instance:
```

Comment:
Be sure to follow the docstring style used in the rest of the project. E.g. for the ``resolve()`` function above:

```
Parameters
----------
sender : class
instance : :class:`.Type` or :class:`.Concept`
```